Gemfile

```diff
@@ -33,6 +33,8 @@ gem 'bootstrap'
 gem 'figaro'
 gem 'pry'
```

Comment:
@naimaseales Move `pry` to the `development, test` group in `Gemfile` to avoid it being run in the production environment. `pry` is only for debugging and best not included in `production` environment.